src/validate/methods/xdr.ts

```diff
@@ -1,7 +1,37 @@
-export const xdr = (value: string) => {
+import { trim } from "@/helpers/trim";
+import { xdr as stellarXDR } from "@stellar/stellar-sdk";
+
+const validateBase64 = (value: string) => {
```

Comment:
I can add this to its own file! but I think so far, this is just used within checking xdr

Reply:
If it's only used here right now, let's keep it here. :+1: